app/config.ts

```diff
@@ -1,7 +1,7 @@
 const CONFIG = {
-  MMO_ECC_ORCHESTRATION_SVC_URL: process.env.MMO_ECC_ORCHESTRATION_SVC_URL,
-  MMO_ECC_REFERENCE_SVC_URL: process.env.MMO_ECC_REFERENCE_SVC_URL,
-  LIMIT_ADD_SPECIES: process.env.LIMIT_ADD_SPECIES
+  MMO_ECC_ORCHESTRATION_SVC_URL: 'http://localhost:5500',
```

Comment:
I don't really think this change is correct

Reply:
This is temporary till I find a working solution. This is currently causing an error and breaking the entire javascript. Main issue is that `process` is undefined in the browser which expected so these values need to be grabbed and used on the server and not added to the browser bundle. I spent time on this and got no idea at the moment and kind of a blocker.

Reply:
But it was correct before... it will be picked up by the .env